src/state/events/selectors.js

```diff
@@ -11,7 +11,7 @@ export const getAllEvents = state => state.events.allEvents;
 export const getAllOldEvents = state => state.events.allOldEvents;
 export const getLoading = state => state.events.loading;
 export const getEventsCounts = state => state.events.eventsCounts;
-export const getTotalEventCounts = state => state.events.totalEventsCounts;
+export const getTotalEventCounts = state => state.events.totalFederalEvents + state.events.totalStateEvents;
```

Comment:
this should be a selector made with createSelector.

Reply:
Updated accordingly in new commit, pushed below